templates/ui_patterns_library/patterns-overview-page.html.twig

```diff
@@ -24,35 +24,32 @@
   } %}
 
   {% for pattern_name, pattern in patterns %}
+    <div class="ecl-u-mv-l ecl-u-border-color-grey-25 ecl-u-border-bottom">
```

Comment:
All of the changes done in this file seem unrelated to the porting the table component.

Reply:
Removed.